.travis.yml

```diff
@@ -30,20 +30,16 @@ matrix:
   include:
     - name: Test Kyuubi w/ -Pspark-3.0
       env:
-        - PROFILE="-Pspark-3.0"
-        - EXCLUDE_TAGS=""
+        - PROFILE="-Pspark-3.0 -Dspark.archive.mirror=https://archive.apache.org/dist/spark/spark-3.1.1 -Dspark.archive.name=spark-3.1.1-bin-hadoop2.7.tgz"
+        - EXCLUDE_TAGS="org.apache.kyuubi.tags.DataLakeTest"
     - name: Test Kyuubi w/ -Pspark-3.1
       env:
         - PROFILE="-Pspark-3.1"
-        - EXCLUDE_TAGS="org.apache.kyuubi.tags.DataLakeTest"
```

Comment:
it is excluded by default, and keep it sync with GA

Reply:
iceberg does not support spark 3.1 yet

Reply:
I see. We need specify it here, otherwise, it will be overwritten by an empty string.